src/limb.rs

```diff
@@ -8,8 +8,9 @@ use {rand, polyfill, c, core};
 
 pub const LIMB_BYTES: usize = (LIMB_BITS + 7) / 8;
 
-/// Stores a positive integer range (e.g. that governing a private key) as
-/// little-endian limbs representing the top end of the range (exclusive).
+/// Stores a positive integer range 0 < x < N (e.g. that governing a private
```

Comment:
I suggest `s/Stores/References` since the limbs aren't stored in the `Range`. Also, I suggest that you clarify the endianness comment a bit: the words are ordered least-significant-word to most-significant-word, and the bytes within the words are in the native endianness.